components/templates/src/global_fns/mod.rs

```diff
@@ -95,18 +96,36 @@ fn compute_file_sha256(mut file: fs::File) -> result::Result<String, io::Error>
     Ok(format!("{:x}", hasher.finalize()))
 }
 
+fn compute_file_sha256_base64(mut file: fs::File) -> result::Result<String, io::Error> {
```

Comment:
I think it will be cleaner to pass the base64 arg to those functions than to duplicate them all.

Reply:
I thought about it but there were already 3 different functions for the 3 possible values of `sha_type` so I opted for keeping it consistent.

Wouldn't it be even better to pass `sha_type` and `base64` to one unique function handling all the possible combinations?

Reply:
> Wouldn't it be even better to pass sha_type and base64 to one unique function handling all the possible combinations?

Even better!